docs/data/pages.ts

```diff
@@ -31,7 +31,17 @@ const pages = [
       { pathname: '/x/react-data-grid/scrolling' },
       { pathname: '/x/react-data-grid/virtualization' },
       { pathname: '/x/react-data-grid/accessibility' },
-      { pathname: '/x/react-data-grid/group-pivot', title: 'Group & Pivot' },
+      {
+        pathname: '/x/react-data-grid/row-grouping',
+        title: 'Group & Pivot',
```

Comment:
Do we really need to group these features under "Group & Pivot"? The master detail doesn't group rows.

Reply:
I just took every section of the current "Group & Pivot" page, but for me, "Master Detail" should go

- in the new "Row" group of pages with pages like "Row Spanning" or "Row Reordering" or "Row Height"

or

- at the root in a standalone page (it's what AG-Grid is doing)


What do you think ?

Reply:
Inside the "row" group seems better, but this can be done later. This new group also affects the pricing page.